docs/authoring-components.md

```diff
@@ -677,31 +679,45 @@ the aforementioned rule, we would also like to provide lint rules to enforce the
   could potentially be localized, etc.) must be referenced by a `strings` static getter.
 - All semantic numbers leveraged by the adapter (timeout lengths, transition durations, etc.) must
```

Comment:
I think that's a typo. I most likely meant _foundation_ there. Looks like it's unrelated to this PR but I'll definitely make that change. 

To answer: the reason we expose numbers is 1) to minimize the amount of magic numbers floating around our codebase 2) A lot of those numbers have something to do with what's happening in CSS, e.g. the amount of time a transition should take. This helps centralize that information and expose it in one place.